.air.conf

```diff
@@ -9,7 +9,7 @@ tmp_dir = "tmp/air"
 # Just plain old shell command. You could use `make` as well.
 cmd = "go build -o bin/easi ./cmd/easi"
 # Binary file yields from `cmd`.
-bin = "bin/easi"
+include_ext = ["go", "sql"]
```

Comment:
Is there a reason we're dropping the `bin` config here? 

Seems like the comment doesn't line up anymore

Reply:
Thanks for catching that Clay, I updated the wrong line on accident, that shouldn't be dropped. It's updated now!